src/github.com/matrix-org/dendrite/federationapi/routing/routing.go

```diff
@@ -129,6 +129,16 @@ func Setup(
 		},
 	)).Methods(http.MethodGet)
 
+	v1fedmux.Handle("/event_auth/{roomID}/{eventID}", common.MakeFedAPI(
+		"federation_get_event_auth", cfg.Matrix.ServerName, keys,
```

Comment:
It seems to me that we've already used this name twice, this will be the third time.
Although I feel `federation_get_event_auth` is correct for this endpoint & we should change metric name for `/state_ids/{roomID}` & `/state/{roomID}` instead.

Reply:
How about `federation_get_state` and `federation_get_state_ids`  ?